### PR TITLE
Add markdown dark mode

### PR DESCRIPTION
**Pull request by DSPaul**
I took the liberty of taking a crack at it myself because it was only like 3 lines of code. Given that the code styles were already grouped in light and dark themes, I decided to just use that so any dark style will return dark markdown and any light style will return light markdown. It would probably be better to swap out the options in the select depending on the whether raw code, markdown or a notebook is chosen but I feel like it works as is for now. What do you think? Feel free to test it out at you own pace, there's no pressure.

### CODE REVIEW

File: embed-v2.js [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
-  const isDarkStyle = !lightStyles.includes(style) && type === 'code';
+  const isDarkStyle = !lightStyles.includes(style);
```
Comment (yusanshi) [inline embed-v2.js:9]: Also add the dark mode for ipynb. So don't need to check `type` here.

File: embed-v2.js [hunk 2]
```diff
@@ -95,6 +95,7 @@
 
   .emgithub-file-dark {
     border: 1px solid #555;
+    background-color: #0d1117;
```
Comment (yusanshi) [inline embed-v2.js:98]: Don't use the `black` color. Instead use the dark color from https://github.com/sindresorhus/github-markdown-css/blob/main/github-markdown-dark.css#L7.

In this way, the whole background is shown in `#0d1117` color.

File: embed-v2.js [hunk 3]
```diff
@@ -350,7 +354,7 @@
 
   if (type === 'markdown' || type === 'ipynb') {
     const loadMarked = typeof marked != "undefined" ? Promise.resolve() : loadScript('https://cdn.jsdelivr.net/npm/marked@4.0.18/marked.min.js');
-    const loadMarkdownStyle = fetch('https://cdn.jsdelivr.net/gh/sindresorhus/github-markdown-css@5.1.0/github-markdown-light.min.css')
+    const loadMarkdownStyle = fetch(`https://cdn.jsdelivr.net/gh/sindresorhus/github-markdown-css@5.1.0/github-markdown-${isDarkStyle ? 'dark' : 'light'}.min.css`)
```
Comment (yusanshi) [inline embed-v2.js:357]: Less code for switching style.

**Comment by yusanshi** [comment]
I like your idea and I have tweaked the code to also support dark mode for the jupyter files. You can check my commits and the comments in GitHub web interface for the details. It that is OK to you, we can merge it!

Thank you and sorry for my late reply :)


## Markdown Dark

image


## Jupyter  Dark

image

**Comment by DSPaul** [comment]
I'm glad you found the time to come back to this, it looks good to me so go ahead and merge!

**Pull request merged**